src/propslib/DelegationConfig.cpp

```diff
@@ -33,7 +33,14 @@ HRESULT _lookupCatalog(PCWSTR extensionName, std::vector<T>& vec) noexcept
 {
     vec.clear();
 
-    auto coinit = wil::CoInitializeEx(COINIT_MULTITHREADED);
+    T useInbox;
+    useInbox.clsid = { 0 };
+    useInbox.name = L"Windows Console Host";

```

Comment:
Yes, but do you have any idea how I would accomplish that?

This is coming out of a LIB that has no .rc. And as far as I know, we can't do .rcs for libraries that roll together.

So I could maybe leave it as a sentinel or constant and let the presenting application flip it out.

Or perhaps there's some API I could use that could go extract the names out of `conhost.exe` in system32 that might vary on locale?

Reply:
Meh. The sentinel thing seems like the "best bet". It would let us MUI it in propsheet and RESW it in settings. I do understand why that sucks though :)

Reply:
(The name in system32 is "Console Window Host" which is pretty ambig.)

Reply:
I used the 0 CLSID as the sentinel and I made propsheet look it up from the string table.